src/styles/CompareCards.ts

```diff
@@ -125,31 +127,29 @@ export const SearchStyles = (mode: string) => {
     component: {
       marginBottom: `${Spacing.xLarge}px`,
     },
+    cancelCompareBtn: {
+      display: 'flex',
+      alignItems: 'center',
+      $nest: {
+        '.cancel-button': {
+          marginRight: `${Spacing.Small}px`,
+        },
+      },
+    },
```

Comment:
can we use this opportunity to style the cancel button directly on the componen rather than with these classes?

Reply:
Yes, that makes sense. Keep forgetting to do that. Will remember for next time. Thanks.